peps/converters.py

```diff
@@ -204,6 +166,15 @@ def get_pep_page(pep_number, commit=True):
         return
 
     pep_content = convert_pep_page(pep_number, open(pep_path).read())
+    pep_ext = '.txt'
+    pep_rst_source = os.path.join(
+            settings.PEP_REPO_PATH, 'pep-{}.rst'.format(pep_number))
+    if os.path.exists(pep_rst_source):
```

Comment:
I know you've just moved the existing code, it would be nice to use this as an opportunity to simplify the code a bit:

```py
pep_rst_source = ...
pep_ext = '.rst' if os.path.exists(pep_rst_source) else '.txt'
```